src/state/routing/useRoutingAPITrade.ts

```diff
@@ -1,13 +1,22 @@
 import { skipToken } from '@reduxjs/toolkit/query/react'
-import { Currency, CurrencyAmount, TradeType } from '@uniswap/sdk-core'
-import { Trade } from '@uniswap/v3-sdk'
+import { Trade } from '@uniswap/router-sdk'
+import { Currency, CurrencyAmount, Token, TradeType } from '@uniswap/sdk-core'
 import ms from 'ms.macro'
 import { useMemo } from 'react'
 import { useBlockNumber } from 'state/application/hooks'
 import { useGetQuoteQuery } from 'state/routing/slice'
+import { useClientSideRouter } from 'state/user/hooks'
 
-import { V3TradeState } from './types'
-import { computeRoutes } from './utils'
+import { TradeState } from './types'
+import { computeRoutes, transformRoutesToTrade } from './utils'
+
+/** Picks required properties of a `Token` to make it serializable */
+const tokenToSerializable = ({ address, chainId, symbol, decimals }: Token) => ({
+  address,
+  chainId,
+  symbol,
+  decimals,
+})
```

Comment:
Do we still need this?

Reply:
just a utility to make tokens serializable in the redux store